src/BusinessService.ts

```diff
@@ -629,6 +666,163 @@ class BusinessService extends Base {
 
     return json as unknown as BillingPageToken;
   }
+
+  /**
+   * List all children LBIDs of a LBID.
+   *
+   * @remarks user has to be the administrator of the business.
+   */
+  async getChildren(id: Business["id"]): Promise<Business[]> {
+    const { apiEndpointAddress, fetch } = this.lensPlatformClient;
+    const url = `${apiEndpointAddress}/businesses/${id}/businesses`;
+    const json = await throwExpected(async () => fetch.get(url), {
+      403: (error) => new ForbiddenException(error?.body?.message),
```

Comment:
Nit. Do we need to add 401 here as well?

Reply:
This is from NestJS guard, default to 403